virttest/libvirt_cgroup.py

```diff
@@ -11,6 +11,7 @@
 
 from virttest import virsh
 from virttest.staging import utils_cgroup
+from virttest import utils_disk
```

Comment:
just one thing the imported modules should be arranged alphabetically according to the clean code instruction 
so the 'utils_disk' could be imported before 'virsh'

Reply:
updated.